calendar.rb

```diff
@@ -0,0 +1,203 @@
+require 'date'
+require_relative 'event_info'
+require 'colorize'
+
+class Calendar
+  def initialize
+    @all_events = {}
+  end
+
+  def put_event date
+    count = 0
+    @all_events.each do |key, value|
+      if key == date
+        count = value.length
+      end
+    end
+    count
+  end
+
+  def show_calendar(month, year)
+    total_days = Date.new(year,month,-1).mday
+    tabs = Date.parse(Time.now.strftime("01/%m/%Y")).wday
```

Comment:
You already have the desired date object at line number 21. That should be used here.